frontend/packages/console-app/src/actions/providers/service-binding-provider.ts

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { K8sResourceKind, referenceFor } from '@console/internal/module/k8s';
+import { useK8sModel } from '@console/shared/src/hooks/useK8sModel';
+import { DeploymentActionFactory } from '../creators/deployment-factory';
+
+export const useCreateServiceBindingProvider = (resource: K8sResourceKind) => {
+  const [kindObj, inFlight] = useK8sModel(referenceFor(resource));
+
+  const deploymentActions = React.useMemo(
+    () => DeploymentActionFactory.CreateServiceBinding(kindObj, resource),
```

Comment:
Shouldn't this be an array?
```suggestion
    () => [DeploymentActionFactory.CreateServiceBinding(kindObj, resource)],
```

Reply:
@rottencandy why should this be an array?

Reply:
it works even if it is not passed as an array. In the ActionsLoader component, all the actions (array of actions/action which is an object) incoming from all the providers are being flattened and put into one single array

Reply:
Didn't know that, thanks